src/sites/amsterdam/components/shared/FormNavigation.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Icon } from '@amsterdam/design-system-react'
+import { Link } from '../../../../shared/components/Link'
+import { ChevronLeftIcon } from '@amsterdam/design-system-react-icons'
+
+const FormNavigation = ({
+  firstPage,
+  lastPage,
+  backlinkUrl,
+}: {
+  firstPage?: boolean
+  lastPage?: boolean
+  backlinkUrl?: string
+}) => (
+  <div className="amsterdam-form-navigation">
+    {!firstPage && backlinkUrl && (
+      <Link to={backlinkUrl} className="signalen-back-link">
```

Comment:
This should be a ‘link that looks like a button’, not sure if we have that already. At least there’s an underline now that shouldn’t be there.